### ISSUE TITLE
jsx emmet_expand_abbreviation

### ISSUE THREAD

**Issue by felixcatto** [emmetio/sublime-text-plugin#138]
I used to expand JSX snippets with `Ctrl+E` hotkey, but seems this doesn't work anymore (v2.2.2). I tried `auto_mark: false` as said in readme, but hotkey still doesn't work, while `Emmet: Enter Abbreviation Mode` works fine with any value in `auto_mark`, but i don't want to use it.

Is there any way to fix `emmet_expand_abbreviation` hotkey?

**Comment by sergeche** [comment]
`auto_mark` option is used for typing and automatically capturing abbreviations. For JSX, you should prefix abbreviations with `<` or disable `jsx_prefix` option to use without prefixes

**Comment by felixcatto** [comment]
Wow, i never got such a fast answers :)
I tried `"jsx_prefix": false,` but this breaks native sublime snippet autocomplete
I found `"tab_expand": false,` option. With it native autocomplete works fine and emmet_expand_abbreviation works fine
but now all text that i type becomes underlined

![Selection_002](https://user-images.githubusercontent.com/6022888/97142943-2428c500-177b-11eb-964b-bd6ea38d9c72.png)
Now i have no idea how to fix it

**Comment by sergeche** [comment]
Underline means Emmet captured abbreviation to expand it with Tab key. To disable underline, you should set `auto_mark` option to false

**Comment by felixcatto** [comment]
But if i disable it, `emmet_expand_abbreviation` stops working :(
All i want to do is to restore hotkey, which i used last 3 years and it worked great. Maybe i can install older version of this package?

**Comment by sergeche** [comment]
There was a bug with abbreviations called by explicit action, fixed in v2.2.3

**Comment by felixcatto** [comment]
in v2.2.3 with
```
  "auto_mark": "stylesheet",
  "jsx_prefix": false,
  "tab_expand": false
```
all works fine. Hotkey working, native autocomplete working, no unnecessary text underline
Thanks :heart:
